app/post.go

```diff
@@ -434,7 +434,7 @@ func (a *App) UpdateEphemeralPost(userId string, post *model.Post) *model.Post {
 
 func (a *App) DeleteEphemeralPost(userId string, post *model.Post) *model.Post {
 	post.Type = model.POST_EPHEMERAL
-	post.UpdateAt = model.GetMillis()
+	post.DeleteAt = model.GetMillis()
```

Comment:
Being a bit picky, both fields are updated to the same when the post is deleted, so the correct code should be

```suggestion
deleteAt = model.GetMillis()
post.UpdateAt = deleteAt
post.DeleteAt = deleteAt
```

Reply:
It worked without it, so I took it out, but sure, I agree it's more explicit.